types/jsonwebtoken/jsonwebtoken-tests.ts

```diff
@@ -160,12 +160,12 @@ decoded = jwt.decode(token, { complete: false, json: false });
 
 decoded = jwt.decode(token, { json: true });
 if (decoded) {
-    // $ExpectType { [key: string]: any; }
+    // $ExpectType JwtPayload
     decoded;
 }
 
 decoded = jwt.decode(token, { complete: true });
 if (decoded) {
-  // $ExpectType { [key: string]: any; }
+  // $ExpectType JwtPayload
```

Comment:
I think this should be `Jwt` no?

Reply:
It should be, based on the provided definition. I'm not sure why it isn't.

Do you get the correct type if you do this?

```ts
// $ExpectType Jwt | null
const decoded2 = jwt.decode(token, { complete: true });
```

Reply:
Huh, yes I do, thanks.